sync/sync-store/src/main/java/com/duckduckgo/sync/store/SyncStore.kt

```diff
@@ -32,11 +33,19 @@ class SyncSharedPrefsStore
 constructor(
     private val sharedPrefsProv: SharedPrefsProvider,
     private val appCoroutineScope: CoroutineScope,
+    private val dispatcherProvider: DispatcherProvider,
 ) : SyncStore {
 
     private val encryptedPreferences: SharedPreferences? by lazy { encryptedPreferences() }
 
-    private val isSignedInStateFlow = MutableStateFlow(isSignedIn())
+    private val isSignedInStateFlow = MutableStateFlow(false)
```

Comment:
Doing it like this may have unintended consequences. Before you the flow was emitting just one state, now we're emitting two.

I had a similar issue and it can be solved using a shared flow. See https://github.com/duckduckgo/Android/pull/3599/files#diff-5dc2abd95191ed6ded4bd91907d7bccfd2eaad8caaf6a22111e4c259ba032bf7R63-R75